Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ministryofjustice/ruby:2.5.1-webapp-onbuild
+FROM ministryofjustice/ruby:2.5.3-webapp-onbuild
```

Comment:
I get them to create this tag yesterday

https://github.com/ministryofjustice/cloud-platform/issues/542